lib/KNXInterfaceManager.js

```diff
@@ -23,10 +26,22 @@ class KNXInterfaceManager extends EventEmitter {
 
     if (ip.isV4Format(localIP)) {
       this.localIPBuffer = ip.toBuffer(ip.address());
+      const interfaces = this.homey.settings.get('interfaces') || [];
 
       this.findKNXInterfaces()
         .catch((error) => {
           this.log('findKNXinterfaces error', error);
+        }).finally(async () => {
+          for (const interface of interfaces) {
```

Comment:
SyntaxError: Unexpected strict mode reserved word
This is my my bad😅 Would `savedInterface` be appropriate?

Reply:
Yes, savedInterface would be appropriate